### ISSUE TITLE
Too long at this step: electrify:  installing electrified dependencies

### ISSUE THREAD

**Issue by eahmedshendy** [arboleya/electrify#14]
Hi, I follow the instructions on my Linux Mint 17.1

``` bash
meteor create --example leaderboard
cd leaderboard
meteor add arboleya:electrify
meteor
```

The process stalled at this:
`I20150924-21:59:15.216(2)? electrify:  installing electrified dependencies`

I'm waiting more than 20 minutes.
And:

``` bash
du -hs .electrify/
16K .electrify/
```

if it is worth to mention, I did this previously:

``` bash
sudo npm install electrify -g
```

**Comment by arboleya** [comment]
You're on OSX, I suppose?

Which version of meteor?

**Comment by eahmedshendy** [comment]
@arboleya 
My Linux is Mint 17.1, and Meteor is 1.2

**Comment by jacquelynjg** [comment]
Hi, I'm experiencing this too. Meteor 1.2.0.1 running on OSX 10.10.3. After:

```
meteor --verbose
```

I got this error:

```
=> App running at: http://localhost:3000/
I20150928-07:31:25.866(8)? electrify:  launching electron
W20150928-07:31:25.871(8)? (STDERR) 
W20150928-07:31:25.872(8)? (STDERR) fs.js:439
W20150928-07:31:25.872(8)? (STDERR)   return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
W20150928-07:31:25.872(8)? (STDERR)                  ^
W20150928-07:31:25.872(8)? (STDERR) Error: ENOENT, no such file or directory '/Users/jacquelyng/hobbi_games/meteor/leaderboard/.electrify/node_modules/electron-prebuilt/path.txt'
W20150928-07:31:25.873(8)? (STDERR)     at Object.fs.openSync (fs.js:439:18)
W20150928-07:31:25.873(8)? (STDERR)     at Object.fs.readFileSync (fs.js:290:15)
W20150928-07:31:25.873(8)? (STDERR)     at Object.<anonymous> (/Users/jacquelyng/hobbi_games/meteor/leaderboard/.electrify/node_modules/electron-prebuilt/index.js:4:21)
W20150928-07:31:25.873(8)? (STDERR)     at Module._compile (module.js:456:26)
W20150928-07:31:25.874(8)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
W20150928-07:31:25.874(8)? (STDERR)     at Module.load (module.js:356:32)
W20150928-07:31:25.874(8)? (STDERR)     at Function.Module._load (module.js:312:12)
W20150928-07:31:25.875(8)? (STDERR)     at Module.require (module.js:364:17)
W20150928-07:31:25.875(8)? (STDERR)     at require (module.js:380:17)
W20150928-07:31:25.875(8)? (STDERR)     at Object.<anonymous> (../meteor/leaderboard/.electrify/node_modules/electron-prebuilt/cli.js:3:16)
```

**Comment by arboleya** [comment]
Hello @eahmedshendy @jacquelynjg 

I've just released a complete revamp version of the project, now it has tests and I'm confident it's way more stable than before.

Can you please update and report back if you still are facing the problem you've mentioned?

Some notes on upgrading:
https://github.com/arboleya/electrify#upgrading

Thanks in advance.

**Comment by arboleya** [comment]
Should be fixed in latest version - 2.0.0.

In case the problem persists, please re-open.
